### ISSUE TITLE
Add longitude and latitude points

### ISSUE THREAD

**Issue by curtispuetz** [vkurup/python-tcxparser#24]
Could add longitude and latitude points just like the project has altitude points with the following methods added to TCXParser:

```
    def latitude_values(self):
        return [float(x.text) for x in self.root.xpath('//ns:Position/ns:LatitudeDegrees', namespaces={'ns': namespace})]

    def longitude_values(self):
        return [float(x.text) for x in self.root.xpath('//ns:Position/ns:LongitudeDegrees', namespaces={'ns': namespace})]
```

**Comment by slackline** [comment]
This would be great, I'm looking to leverage tcxparser for converting `.tcx` to `.gpx` files.

**Comment by vkurup** [comment]
Yes, looks like @janpipek added Latitude and Longitude in tuples returned by position_values() in #23, so perhaps I could close this issue now.

**Comment by vkurup** [comment]
Closed by #23
